delivery_recipe/business_continuity_plan.md

```diff
@@ -0,0 +1,134 @@
+
+## Plan Review
+
+The plan is reviewed annually (as a minimum). The next review date is planned for June 2017.
+
+## Plan Testing
+
+The plan is due to be tested before the next review in June 2017.
+
+# Plan Activation
+
+## Circumstances
+
+The plan will be activated in response to an incident causing significant disruption to normal service delivery, particularly in the delivery of projects or access to data. Examples of circumstances triggering activation of this plan include:
+
+* Loss of critical systems
+* Sudden loss of key staff or skills
+* Loss or theft of mobile devices (e.g. laptop, phone)
+* Loss, including suspected loss or unauthorised access, of data
+
+## Responsibility for Activation
+
+A member of our Business Continuity Team (BCT) will activate and initiate this plan:
+
+* Steve Parks, CEO
+* Steve Hunton, COO
+* Joe Baker, Lead Developer
+
+# Incident Management
+
+## Purpose of the Incident Management Phase
+
+* Protect vital assets e.g. equipment, data, reputation, clients
+* Ensure necessary communication takes place
+* Support the Business Continuity phase
+* Support the Recovery and Resumption phase
+
+## Actions to Protect Vital Assets
+
+The following actions will be taken to protect our vital assets:
+
+1. Whoever identifies the incident should record and report details of the incident. If possible, email these to hello@weareconvivio.com
+2. Inform members of the Business Continuity Team (see above)
+3. The BCT assess the impact of the incident to agree response and next steps
+4. The BCT team communicates the incident to other members of the team and customer as agreed in their next steps
+
+## Communication Actions
+
+In the event of an incident and this plan being initiated, the BCM team should be contacted. Other people within the team or within the client base may also be contacted depending on the details of the incident.
+
+# Incident Scenarios
+
+Following are a list of possible incident scenarios that could risk business continuity and the key steps to be taken in order to manage the risks.
+
+## Loss or Theft of Mobile Device
+
+* The member of staff reports details of the loss or theft to hello@weareconvivio.com or calls the main phone number 020 3875 3438
+* The incident is reported to the support team
+* Details are recorded in the [Incident Management Report](https://docs.google.com/spreadsheets/d/1ZaHs2A5mMlN3m4aXX_lCGMBEYTMNAknwHPpsJzbCJDI/edit#gid=0)
+* GitHub account is suspended
+* Google account is suspended
+* RingCentral account is suspended
+* Xero account is suspended
+* Slack account is suspended
+* A location trace is attempted using Apple's Find My Phone
+* Details are reported to the Police
+
+## Loss of Critical Systems
+
+Critical systems include any that affect our ability to do our jobs. These would include Google Drive, Slack and GitHub
```

Comment:
what about if internet connectivity is lost in general? We can mention that our free range nature means we have different internet service providers in different geographic locations, plus we also have mobile broadband connections - so would only be affected in the event of a national outage of everything.